style.css

```diff
@@ -1 +1,134 @@
-/*  Add your styling here */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.header-content {
+  display: grid;
+  grid-template-columns: 45% 55%;
+  background-color: rgb(12, 228, 113);
+  padding: 20px;
+  border-bottom: 1px solid #eee;
+}
+.logo-img {
+  grid-row: 1 / -1;
+  margin-left: 10px;
+}
+.logo-img img {
+  display: grid;
```

Comment:
So that the image fills its grid cell and gets bigger / smaller as the viewport changes width, try removing `display: grid` from this element, and setting `width: 100%`